scripts/skills/perks/perk_rf_ghostlike.nut

```diff
@@ -51,8 +107,84 @@ this.perk_rf_ghostlike <- ::inherit("scripts/skills/skill", {
 			if (numAllies >= numEnemies)
 			{
 				_properties.IsImmuneToZoneOfControl = true;
```

Comment:
This part of the effect does not check whether the global conditions of this perk are active.